src/modules/markets/selectors/tags.js

```diff
@@ -1,46 +1,49 @@
-import memoizerific from 'memoizerific';
+import { createSelector } from 'reselect';
+import store from 'src/store';
+import { selectSelectedTagsState } from 'src/select-state';
+import { selectFilteredMarkets } from '../../markets/selectors/markets-filtered';
```

Comment:
Does proxyquire use the path aliases as well?  Generally I've had to keep things using relative paths if I need to stub them with proxyquire -- maybe I am not specifying the paths correctly?